util-core/src/main/scala/com/twitter/util/Future.scala

```diff
@@ -104,6 +104,15 @@ object Future {
   def const[A](result: Try[A]): Future[A] = new ConstFuture[A](result)
 
   /**
+   * Creates a satisfied `Future` from a [[scala.util.Try]].
+   *
+   * @see [[value]] for creation from a constant value.
+   * @see [[apply]] for creation from a `Function`.
+   * @see [[exception]] for creation from a `Throwable`.
+   */
+  def const[A](result: scala.util.Try[A]): Future[A] = const(Try.fromScalaTry(result))
```

Comment:
is this sugar necessary? i'd prefer pointing people at `Try.fromScalaTry` in the `const` scaladocs instead.